cmd/ui.go

```diff
@@ -111,31 +112,25 @@ func modifyAndPrintBar(bar *pb.ProgressBar, options ...pb.ProgressBarOption) {
 // Print execution description for both cloud and local execution.
 // TODO: Clean this up as part of #1499 or #1427
 func printExecutionDescription(
-	execution, filename, output string, conf Config, et *lib.ExecutionTuple,
-	execPlan []lib.ExecutionStep, collectors []lib.Collector,
+	execution, filename, outputOverride string, conf Config, et *lib.ExecutionTuple,
+	execPlan []lib.ExecutionStep, outputs []output.Output,
 ) {
 	fprintf(stdout, "  execution: %s\n", ui.ValueColor.Sprint(execution))
 	fprintf(stdout, "     script: %s\n", ui.ValueColor.Sprint(filename))
 
-	if execution == "local" {
-		out := "-"
-		link := ""
-
-		for idx, collector := range collectors {
-			if out != "-" {
-				out = out + "; " + conf.Out[idx]
-			} else {
-				out = conf.Out[idx]
-			}
-
-			if l := collector.Link(); l != "" {
-				link = link + " (" + l + ")"
-			}
+	var outputDescriptions []string
+	switch {
+	case outputOverride != "":
+		outputDescriptions = []string{outputOverride}
+	case len(outputs) == 0:
+		outputDescriptions = []string{"-"}
+	default:
+		for _, out := range outputs {
+			outputDescriptions = append(outputDescriptions, out.Description())
 		}
-		fprintf(stdout, "     output: %s%s\n", ui.ValueColor.Sprint(out), ui.ExtraColor.Sprint(link))
-	} else {
-		fprintf(stdout, "     output: %s\n", ui.ValueColor.Sprint(output))
 	}
+
+	fprintf(stdout, "     output: %s\n", ui.ValueColor.Sprint(strings.Join(outputDescriptions, ";")))
```

Comment:
Nit: maybe add a space after the semi-colon and use a comma instead. Here's what it currently looks like:

```
output: json(test.json);influxdb (http://127.0.0.1:8086)
```

I guess a space between `json` and `(test.json)` would be good too.

Reply:
Resolved by https://github.com/loadimpact/k6/commit/ac4a3f98155cd370e0e8be895f971aee2efc440f